app/services/users/profile.rb

```diff
@@ -80,7 +80,6 @@ def profile
         verified: user.loa3?,
         sign_in: user.identity.sign_in,
         authn_context: user.authn_context,
-        inherited_proof_verified: user.inherited_proof_verified,
```

Comment:
I looked at FE, I don't see anything using `inheritedProofVerified`, so this should be good. However, since you're updating the serializer that FE gets, can you also update the FE code to remove this field? (I think it's just a number of test files that include the `inherifedProofVerified` serialized attribute)